internal/gcs/container.go

```diff
@@ -248,8 +228,9 @@ func (c *Container) Wait() error {
 }
 
 func (c *Container) waitBackground() {
-	ctx, span := oc.StartSpan(context.Background(), "gcs::Container::waitBackground")
+	_, span := oc.StartSpan(context.Background(), "gcs::Container::waitBackground")
```

Comment:
why are we keeping this span but not the others on a `Container`?

Reply:
all the other `Container` operations eventually end up with a `gcs` and then `bridge call`, so they have spans being created for them. 
This is the only one that doesn't. Plus this is a long running call that is launched asynchronously, so we dont have any other way of tracking it, which i felt made it worth of a dedicated span